src/components/pages/farmer-produce/list/index.tsx

```diff
@@ -109,66 +98,60 @@ function FarmerListPageComponent() {
 
   const onFarmerUpdate = (props) => {
     onToggle();
-    actions.updateFarmer(props);
+    actions.updateFarmerProduce(props);
   };
 
-  const loadingColumns = Array.from({ length: 5 }).map((_, index) => (
-    <Th key={index}>
-      <Skeleton height="20px" startColor="gray.200" endColor="gray.400" />
-    </Th>
-  ));
-
-  const loadingRows = Array.from({ length: 5 }).map((_, index) => (
-    <Tr key={index}>
-      {loadingColumns.map((cell, cellIndex) => (
-        <Td key={cellIndex}>
-          <Skeleton height="20px" startColor="gray.200" endColor="gray.400" />
-        </Td>
-      ))}
-    </Tr>
-  ));
-
   return (
     <Box>
       <PageHeading actions={<ActionButtons />}>🚜 Farmer Produce </PageHeading>
```

Comment:
This static content/headigs etc should be in a translation file and then accessed here instead of hardcoding . Other such occurrences in this PR should also be put in translation file as far as possible.